Sources/Services/AWSS3/S3Client.swift

```diff
@@ -5023,3 +5023,57 @@ extension S3Client: S3ClientProtocol {
     }
 
 }
+
+extension S3Client {
+    public func presignURLForGetObject(input: GetObjectInput, expiration: Foundation.TimeInterval) async throws -> Foundation.URL {
```

Comment:
All of these public `presign...()` methods should have documentation comments.

Reply:
Added documentation generation to codegen.